code/modules/mob/thermoregulation.dm

```diff
@@ -0,0 +1,26 @@
+/mob/living/proc/burn_calories(var/amount,var/forceburn = 0)
+	if(forceburn && ticker && ticker.hardcore_mode)
+		forceburn = 0
+	if(nutrition - amount > 0 || forceburn)
+		var/heatmodifier = 0.7
+		nutrition = max(nutrition - amount,0)
+		if((M_FAT in mutations))
+			heatmodifier = heatmodifier*2
+		bodytemperature += amount * heatmodifier
```

Comment:
Theres still several places that burn a LOT of calories at once that this will fuck people over